Cython/Compiler/Optimize.py

```diff
@@ -2580,26 +2581,56 @@ def _handle_simple_function_set(self, node, function, pos_args):
                 is_temp=node.is_temp,
                 py_name="set"))
 
-    PyFrozenSet_New_func_type = PyrexTypes.CFuncType(
-        Builtin.frozenset_type, [
-            PyrexTypes.CFuncTypeArg("it", PyrexTypes.py_object_type, None)
-        ])
-
+    # SimpleCallNode is replaced
```

Comment:
This is a misleading comment at this point because it describes something that applies to pretty much all methods in this class but makes it seem like it's specific to this one.
```suggestion
```

Reply:
Thanks. I re-phased my comment